faucet.go

```diff
@@ -262,7 +266,7 @@ func (l *lightningFaucet) sweepZombieChans(timeCutOff time.Time) {
 	openChanReq := &lnrpc.ListChannelsRequest{}
 	openChannels, err := l.lnd.ListChannels(ctxb, openChanReq)
 	if err != nil {
-		log.Println("unable to fetch open channels: %v", err)
+		log.Printf("unable to fetch open channels: %v", err)
```

Comment:
log.Printf calls are not adding the final line break, so the logs are unreadable.

Ideally, you should change this to use slog instead of log so that the generated logs are in line with other decred tools.